QUEST_LV_0100.tsv

```diff
@@ -4460,20 +4460,20 @@ QUEST_LV_0100_20150317_004455	{memo X}Obtained the poison of Woodfung
 QUEST_LV_0100_20150317_004456	You are all ready to lure Saugas. Talk to Eta who is confident.
 QUEST_LV_0100_20150317_004457	{memo X}Put to stone with the vigor on Alkune Stairway.
 QUEST_LV_0100_20150317_004458	Eta told you to put the stone on Alkune Stairway to lure Saugas. Is he trying to defeat Saugas by himself?
-QUEST_LV_0100_20150317_004459	Saugas took the stone and ran away! Return to Eta and ask him.
-QUEST_LV_0100_20150317_004460	Defeat Saugas
-QUEST_LV_0100_20150317_004461	Eta can't believe that Saugas ran away with the stone. Talk to Eta again.
-QUEST_LV_0100_20150317_004462	Lure Saugas from the right side of Alkune Stairway
-QUEST_LV_0100_20150317_004463	Eta is saying that it can't be true that Saugas just ran away with the stone. Let's put the stone on the deepest place of Long Bridge Valley where Saugas often shows up.
-QUEST_LV_0100_20150317_004464	Saugas didn't show up, but only Sauga appeared. Suggest to Eta to give up and go back to the camp.
+QUEST_LV_0100_20150317_004459	The Saugas took the stone and ran away! Return to Eta and ask him.
+QUEST_LV_0100_20150317_004460	Defeat Sauga
+QUEST_LV_0100_20150317_004461	Eta can't believe that the Saugas ran away with the stone. Talk to Eta again.
+QUEST_LV_0100_20150317_004462	Lure the Saugas from the right side of Alkune Stairway
+QUEST_LV_0100_20150317_004463	Eta is saying that it can't be true that the Saugas just ran away with the stone. Let's put the stones on the deepest place of Long Bridge Valley where Saugas often shows up.
+QUEST_LV_0100_20150317_004464	Saugas didn't show up, but only the Saugas appeared. Suggest to Eta to give up and go back to the camp.
```

Comment:
Saugas didn't show up, but only the Saugas appeared <- Doesn't make sense.

There are 2 monsters: Saugas and Sauga. 

"Saugas didn't show up, but only the Sauga appeared." ?